modules/luci-mod-status/luasrc/controller/admin/status.lua

```diff
@@ -6,6 +6,7 @@ module("luci.controller.admin.status", package.seeall)
 
 function index()
 	entry({"admin", "status", "overview"}, template("admin_status/index"), _("Overview"), 1)
+	entry({"admin", "status", "overview", "kick"}, post("act_kick")).leaf = true
```

Comment:
Please rename the function to `action_disconnect` and the url part to `disconnect`.